draft-ietf-teep-architecture.md

```diff
@@ -1347,6 +1347,33 @@ still a threat present in allowing a potentially compromised TAM to install
 any TA it wants; for example, a known-compromised TA or an explicitly
 malicious TA.
 
+The device manufacturer may want to include a notification support that can allow
+Device Administrators be notified for incidents upon detection with some mitigation 
+methods as suggested above.
+
+## Hostile TAM
+
+There are also threats of hostile or abusive TAMs where a TAM turns to act out of 
+expectation of Device Administrators, for example, pushing out TAs that contain 
+some data collection or use users' device resources for distributed jobs for a TAM. 
+To mitigate this threat, TEEs and Device Administrators have several options for 
+detecting and mitigating a hostile TAM:
+
+1. ACLs: Apply an ACL to TAMs, making explicit which TAs the TAM is permitted to install. 
+This limits the scope of a hostile TAM's impact.
+2. Trust Anchor update: when a Trust Anchor entry in the TEE Trust Anchor list is specific to 
+the hostile TAM, a Trust Anchor update process may be used to remove the Trust Anchor as a way 
+to stop the TAM for installing TAs to the TEE.
+3. TAM certificate revocation: the certificate issuing authority of the TAM certificate is 
+requested to revoke the TAM's certificate. The TEEP Agents that supports certificate revocation 
+status check will be able to stop the TAM, and further decide to remove those hostile TAs from that TAM.
+4. TA removal by the Device Administrator upon detection of a hostile TAM and any abusive TA.
+The detection cab be the result of 1) above, and some other out-of-band detection technique.
+
+The device manufacturer may want to include a notification support that can allow
+Device Administrators be notified for incidents upon detection with some mitigation 
+methods as suggested above.
```

Comment:
Seems like a duplicate of line 1350?

Reply:
Yes, I added this to mean notification should be supported for detection of a hostile TAM too. Do you prefer we better leave it out and someway the notification suggestion in the other Threat item is sufficient to cover?

Reply:
Since we already went through last call, I would avoid any controversial topics if at all possible.   This is one.

Reply:
Brendon expanded the "Compromised TAM" section a lot. Shall we pull out some?

The "hostile TAM" comment is from Ben Schwarts, asking for notification to an admin. So this tries to address it. Notification is kind of out of scope, which means monitoring capability first before alert. Monitoring of a TAM and TEE is much extra.

Let me remove notification sentences in both. How about mitigations that Brendon added and my revised list above? Please review again. Thanks.